clang/include/clang/CodeGen/CodeGenABITypes.h

```diff
@@ -126,6 +126,12 @@ uint16_t getPointerAuthDeclDiscriminator(CodeGenModule &CGM, GlobalDecl GD);
 uint16_t getPointerAuthTypeDiscriminator(CodeGenModule &CGM,
                                          QualType FunctionType);
 
+/// Return a signed constant pointer.
+llvm::Constant *getConstantSignedPointer(CodeGenModule &CGM,
+                                         llvm::Constant *pointer, unsigned key,
```

Comment:
We should capitalize the names to match the other code (both in the .h and the .cpp), while we're here